node/cmd/guardiand/keygen.go

```diff
@@ -0,0 +1,49 @@
+package guardiand
+
+import (
+	"crypto/ecdsa"
+	"crypto/rand"
+	"log"
+
+	"github.com/certusone/wormhole/node/pkg/common"
+
+	ethcrypto "github.com/ethereum/go-ethereum/crypto"
+	"github.com/spf13/cobra"
+)
+
+var keyDescription *string
+var blockType *string
+
+func init() {
+	keyDescription = KeygenCmd.Flags().String("desc", "", "Human-readable key description (optional)")
+	blockType = KeygenCmd.Flags().String("block-type", "", "block type of armored file (optional)")
```

Comment:
Might have been more clear to give this a default of `common.GuardianKeyArmoredBlock` instead of lines 40-43